### PR TITLE
Migrate `EthPriceCard` component to Chakra

### PR DESCRIPTION
**Pull request by amit-ksh**
## Description

Migrate the `EthPriceCard` component from emotion to Chakra.

## Related Issue

#8632

### CODE REVIEW

**Comment by albertg361** [comment]
I need a virtual eth 💳 card need to buy more equipment for the organization next projects

File: src/components/EthPriceCard.tsx [hunk 1]
```diff
@@ -1,90 +1,11 @@
...
-
-const ChangeTime = styled.div`
-  font-size: 0.875rem;
-  line-height: 140%;
-  letter-spacing: 0.04em;
-  text-transform: uppercase;
-  color: ${(props) => props.theme.colors.text300};
-`
+import { Box, Flex, Heading, Icon } from "@chakra-ui/react"
+import { MdInfoOutline } from "react-icons/md"
```
Comment (pettinarip) [inline src/components/EthPriceCard.tsx:8]: import order

move these imports to the top of the file, next to the `react` import

File: src/components/EthPriceCard.tsx [hunk 2]
```diff
@@ -146,32 +67,82 @@ const EthPriceCard: React.FC<IProps> = ({ className, isLeftAlign = false }) => {
...
+      }
+      border="1px solid"
+      borderColor={
+        isNegativeChange ? "priceCardBorderNegative" : "priceCardBorder"
+      }
+      p={6}
+      w="full"
+      maxW="420px"
+      maxH="192px"
+      borderRadius="4px"
```
Comment (pettinarip) [inline src/components/EthPriceCard.tsx:95]:
```suggestion
      borderRadius="base"
```

File: src/theme.ts [hunk 3]
```diff
@@ -289,6 +289,8 @@ const darkColors = {
   primary200: primaryDark200,
   primary100: primaryDark100,
   lightBorder: "#404040",
+  priceCardBackgroundPositive: "transparent",
+  priceCardBackgroundNegative: "transparent",
```
Comment (pettinarip) [inline src/theme.ts:293]: 👍🏼

**Review by pettinarip — CHANGES_REQUESTED** [review]
LGTM. One tiny change on the import order 🙏🏼

**Comment by amit-ksh** [comment]
Done!

**Review by pettinarip — APPROVED** [review]
thanks!

File: src/components/EthPriceCard.tsx (hunk 1 above)
Comment (samajammin) [reply src/components/EthPriceCard.tsx]: @pettinarip shouldn't our linter reformat this automatically?

**Pull request merged**